libs/application/templates/transport-authority/change-operator-of-vehicle/src/utils/getSelectedVehicle.ts

```
@@ -35,16 +32,15 @@ export const getSelectedVehicle = (
 
   const index = parseInt(vehicleIndex, 10)
 
-  if (
-    currentVehicleList?.vehicles &&
-    currentVehicleList.vehicles[index] &&
-    !Object.isFrozen(currentVehicleList.vehicles[index])
-  ) {
+  const vehicle =
+    currentVehicleList?.vehicles && currentVehicleList.vehicles[index]
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)